packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/collection.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@ciscospark/test-helper-chai';
+import MeetingCollection from '@webex/plugin-meetings/src/meetings/collection';
+
+describe('plugin-meetings', () => {
+  describe('meetings collection', () => {
+    beforeEach(() => {
+      MeetingCollection.meetings = {};
+    });
+    describe('#get', () => {
+      beforeEach(() => {
+        MeetingCollection.meetings.test = 'test';
+      });
+      it('should get the meeting', () => {
+        assert.equal(MeetingCollection.get('test'), 'test');
+      });
+    });
+    describe('#set', () => {
+      it('should set the meeting', () => {
+        assert.equal(Object.keys(MeetingCollection.meetings).length, 0);
+        MeetingCollection.set({id: 'test'});
+        assert.equal(Object.keys(MeetingCollection.meetings).length, 1);
+      });
+    });
+    describe('#remove', () => {
+      beforeEach(() => {
+        MeetingCollection.meetings.test = 'test';
+      });
+      it('should get the meeting', () => {
+        assert.equal(Object.keys(MeetingCollection.meetings).length, 1);
+        assert.equal(MeetingCollection.remove('test'), 'test');
+        assert.equal(Object.keys(MeetingCollection.meetings).length, 0);
+      });
+    });
+    describe('#getAll', () => {
+      beforeEach(() => {
+        MeetingCollection.meetings.test = 'test';
```

Comment:
using 'test' is problematic, use like id1 or id2, or value1 value2, or pmr pmr1, pmr2 etc